src/resolve-script-name.js

```diff
@@ -0,0 +1,28 @@
+import {get} from 'lodash'
+
+export default resolveScriptName
+
+function resolveScriptName(scriptToFind, scriptsConfig) {
```

Comment:
The best solution in my opinion would to be contribute to `prefix-matcher`, implement returning property (whole chain, not only the deepest) and then we could extend `getScriptToRun` with it.